examples/embeddable_examples/public/react_embeddables/data_table/data_table_react_embeddable.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { EuiScreenReaderOnly } from '@elastic/eui';
+import { css } from '@emotion/react';
+import { CellActionsProvider } from '@kbn/cell-actions';
+import { CoreStart } from '@kbn/core-lifecycle-browser';
+import { ReactEmbeddableFactory } from '@kbn/embeddable-plugin/public';
+import { i18n } from '@kbn/i18n';
+import { KibanaContextProvider } from '@kbn/kibana-react-plugin/public';
+import { Storage } from '@kbn/kibana-utils-plugin/public';
+import {
+  apiPublishesDataViews,
+  initializeTimeRange,
+  initializeTitles,
+  useBatchedPublishingSubjects,
+} from '@kbn/presentation-publishing';
+import { KibanaRenderContextProvider } from '@kbn/react-kibana-context-render';
+import { DataLoadingState, UnifiedDataTable, UnifiedDataTableProps } from '@kbn/unified-data-table';
+import React, { useEffect } from 'react';
+import { BehaviorSubject } from 'rxjs';
+import { StartDeps } from '../../plugin';
+import { apiPublishesSelectedFields } from '../field_list/publishes_selected_fields';
+import { DATA_TABLE_ID } from './constants';
+import { initializeDataTableQueries } from './data_table_queries';
+import { DataTableApi, DataTableProvider, DataTableSerializedState } from './types';
+
+export const isDataTableProvider = (api: unknown): api is DataTableProvider => {
```

Comment:
Looks like `isDataTableProvider` is not used any where and can be removed

Reply:
Good call, removed in e19133b6ad3